android/hybridcommon/build.gradle.kts

```diff
@@ -49,8 +61,10 @@ android {
 }
 
 dependencies {
-    api(libs.purchases)
-    api(libs.purchases.amazon)
+    "bc8Api"(libs.purchases.bc8)
+    "bc8Api"(libs.purchases.amazon.bc8)
+    "bc7Api"(libs.purchases.bc7)
+    "bc7Api"(libs.purchases.amazon.bc7)
```

Comment:
What is the effect of these "bcXAPI" names? Are they later referenced when importing the dependencies?

Reply:
Yeah, this is used by kotlin gradle dsl to use the correct dependency on each flavor. So `"bc8Api"` means that the dependency will only be included as `api` (this means that it's exposed to consumers) in the `bc8` flavor. Gradle is later smart enough to know what dependencies it needs to use for each flavor.